src/Interface/Tabs/Map.as

```diff
@@ -405,6 +405,14 @@ class MapTab : Tab
         UI::SetPreviousTooltip("Mood");
         UI::Text(Icons::Money + " \\$f77" + m_map.DisplayCost);
         UI::SetPreviousTooltip("Coppers cost");
+
+        /* Inlined styles, because this is the only place where a gold button is needed */
+        UI::PushStyleColor(UI::Col::Button, UI::HSV(0.12f, 1, 0.7f));
+        UI::PushStyleColor(UI::Col::ButtonHovered, UI::HSV(0.12f, 1.1f, 0.8f));
+        UI::PushStyleColor(UI::Col::ButtonActive, UI::HSV(0.12f, 1.2f, 0.9f));
```

Comment:
Can you use `UI::ColoredButton()` or something like that instead of pushing style colors?
The method is listed in src/Utils/UI